packages/diagram/src/module/diagrams/content/uml/sequence/lostFoundMessage.ts

```diff
@@ -0,0 +1,103 @@
+import { fun, id, InterpreterModule, numberType, optional } from "@hylimo/core";
+import { SCOPE } from "../../../../base/dslModule.js";
+
+/**
+ * Lost and found message are synonyms, so should use the same function.<br>
+ * However, we cannot assign it within the expressions to not pollute the global namespace.
+ *
+ * @param type the type of message to use as class name for the generated element
+ * @return the syncscript function string for the given message type
+ */
+function message(type: string): string {
+    return `
+                distance = args.distance ?? scope.externalMessageDistance
```

Comment:
pls decrease indentation for the content and increase indentation for the last line